test/y2network/config_reader/sysconfig_test.rb

```diff
@@ -112,13 +112,23 @@
 
   describe "#forward_ipv4?" do
     it "returns true when IPv4 forwarding is allowed" do
-      expect(reader.send(:forward_ipv4?)).to be_truthy
+      expect(reader.send(:forward_ipv4?)).to be true
+    end
+
+    it "returns false when IPv4 forwarding is disabled" do
+      allow(Yast::SCR).to receive(:Read).and_return("0")
+      expect(reader.send(:forward_ipv4?)).to be false
     end
   end
 
   describe "#forward_ipv6?" do
+    it "returns true when IPv6 forwarding is allowed" do
+      allow(Yast::SCR).to receive(:Read).and_return("1")
+      expect(reader.send(:forward_ipv4?)).to be true
```

Comment:
I guess you meant `forward_ipv6`. On the other hand, you are testing a private method and that's wrong. I think that you should check whether the value returned by `#config` contains the right values. What do you think?